apps/frontend/src/libs/components/date-input/styles.module.css

```
@@ -0,0 +1,144 @@
+.calendar:global(.react-calendar) {
+	display: flex;
+	flex-direction: column;
+	padding: 12px;
+}
+
+:global(.react-calendar__navigation) {
```

with the past comment I meant to add .calendar to all places with use :global selector
```suggestion
.calendar:global(.react-calendar__navigation) {
```